app/main.py

```diff
@@ -1,6 +1,32 @@
+import os
+import json
+import requests
+
+
+URL = "https://api.weatherapi.com/v1/current.json"
+FILTERING = "Paris"
```

Comment:
as for me, better to name it city, not filtering

Reply:
I got it, you have an example in checklist with such naming. It's totally fine 🙌